app/components/CreateQuestionContainer.tsx

```diff
@@ -113,7 +108,7 @@ export default function CreateQuestion({
     setButtonDisabled(true);
     try {
       const txnResponse = await createQuestion.writeAsync({
-        args: [fileUrl, BigNumber.from("10")],
+        args: [fileUrl],
```

Comment:
not sure but I think our current polygon deployments have this extra param no?  Does it break when leaving this param out?